runtime/moonbase/src/weights/frame_system_extensions.rs

```diff
@@ -114,4 +114,9 @@ impl<T: frame_system::Config> frame_system::ExtensionsWeightInfo for WeightInfo<
 			.saturating_add(T::DbWeight::get().reads(1_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 	}
+
+    fn weight_reclaim() -> Weight {
+        // TODO: regenerate this file
+        <()>::weight_reclaim()
```

Comment:
Lets try to generate this locally, just to be sure the benchmarks are not broken